Articles/Blog/what-is-an-inclusive-minute.md

```diff
@@ -0,0 +1,143 @@
+
+People who have trouble empathizing with some dimensions of inclusion can often identify with other dimensions.
+When they do, that can lead to broadening their perspectives more generally.
+
+#### We will all experience the impairment dimension
+
+Towards that end, it is worth considering that we are all likely to experience the disability or impairment dimension of inclusion [at some point](https://accessibility.iu.edu/docs/Types_of_impairments.pdf) in our lives.
+As a trivial example, when participating via an audio-only modality in a video teleconference where visual content is being shared, you will experience situational visual impairment.
+Because of the situation, you are not able to see information which may be critical for you to fully participate and contribute.
+Likewise, if you are just home from a tough visit with the dentist, you may experience temporary speaking impairment.
+If you are recovering from an accident and need to use a wheel chair, you will experience temporary mobility impairment.
+
+#### Recommended logistics
+
+When preparing an inclusive minute (IM), there is no need for any formalism.
+In fact, the less formality the better because it lowers the bar for anyone considering leading an inclusive minute.
+Its best to even discourage the creation of slides but if an IM leader really wants to create a slide to share, they should be asked to keep it to a single slide.
+Leadership doesn't require a lot.
+Its useful to keep remarks brief...maybe just a minute or two at the most.
+The word "minute" in the title of the practice doesn't necessarily mean it should be an actual minute...just a small amount of time so its easy to add to regular meeting agendas.
+Depending on a team's enthusiasm for follow up discussion on any given topic, in reality, an IM will typically take a *few* minutes.
+
+Its useful to consider the appropriate cadence for the inclusive minute practice as well.
+If your team meets once a week, identifying new IM topics for every meeting may become a challenge.
+Doing an IM every other meeting or the first meeting of each month may be best.
+If your team meets monthly, having an inclusive minute with every meeting may make the most sense.
+
+Some teams prefer to have inclusive minute at the end of their meetings and others prefer the beginning.
+There are advantages and disadvantages to either.
+At the beginning, follow-up dialog can run longer than expected putting the meeting leader in the position of having to cut-off conversation to get to the rest of the agenda.
+But, having the exercise at the beginning of the meeting also ensures it actually happens.
+It shows commitment.
+At the end of the meeting, unless the team is committed to reserving time for the activity, it can feel rushed and/or like an afterthought.
```

Comment:
```suggestion
At the end of the meeting, unless the team is committed to reserving time for the activity, it can feel rushed or like an afterthought.
```